### ISSUE TITLE
services "sealed-secrets-controller" not found

### ISSUE THREAD

**Issue by bidandows** [bitnami-labs/sealed-secrets#758]
**Which component**:
The name (and version) of the affected component (controller or kubeseal)
Helm chart ersion 2.1.3 and sealed-secrets-v0.17.3
**Describe the bug**
I got error message after instaling the controller and kubeseal binary 

**To Reproduce**
- 
1. Installed Helm chart via terraform
2. Downloaded kubeseal-0.17.3-linux-amd64.tar.gz file and installed it
3. rune kubeseal secret-source-file.yaml destination-sealed-secret.yaml
4. I got this error message `error: cannot get sealed secret service: services "sealed-secrets-controller" not found`

**Expected behavior**
should seal secret

**Version of Kubernetes**:

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.4", GitCommit:"e6c093d87ea4cbb530a7b2ae91e54c0842d8308a", GitTreeState:"clean", BuildDate:"2022-02-16T12:38:05Z", GoVersion:"go1.17.7", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"21+", GitVersion:"v1.21.5-eks-bc4871b", GitCommit:"5236faf39f1b7a7dabea8df12726f25608131aa9", GitTreeState:"clean", BuildDate:"2021-10-29T23:32:16Z", GoVersion:"go1.16.8", Compiler:"gc", Platform:"linux/amd64"}
WARNING: version difference between client (1.23) and server (1.21) exceeds the supported minor version skew of +/-1
```

**Additional context**

**Comment by surajnarwade** [comment]
can you check `kubectl get service sealed-secrets-controller -n kube-system` ?

**Comment by bidandows** [comment]
yes here it is `sealed-secrets                      ClusterIP   172.20.233.150   <none>        8080/TCP        43h`

**Comment by bidandows** [comment]
issue solved, It's not really an issue maybe the documentation needs to be updated.
This is how I managed to to use kubeseal `kubeseal --controller-name=sealed-secrets --controller-namespace=kube-system --format yaml <secret.yaml>sealed.yaml`

**Comment by hpl002** [comment]
> issue solved, It's not really an issue maybe the documentation needs to be updated. This is how I managed to to use kubeseal `kubeseal --controller-name=sealed-secrets --controller-namespace=kube-system --format yaml <secret.yaml>sealed.yaml`

This screws me every time.

TLDR for the next guy:
You may have to specify controller name and namespace. Be conscious of defaults :) 

```
      --controller-name string           Name of sealed-secrets controller. (default "sealed-secrets-controller")
      --controller-namespace string      Namespace of sealed-secrets controller. (default "kube-system")
```
